app/assets/javascripts/admin/modules/edition-form.js

```diff
@@ -37,5 +38,32 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  EditionForm.prototype.toggleWorldNewsStoryVisibility = function () {
+    var form = this.module
+
+    var select = form.querySelector('#edition_news_article_type_id')
+
+    if (!select) { return }
+
+    var container = form.querySelector('.edition-form--locale-fields')
+    var localeCheckbox = container.querySelector('#edition_create_foreign_language_only-0')
+    var localeSelect = container.querySelector('#edition_primary_locale')
+
+    if (select.value !== '4') {
```

Comment:
Maybe add a comment or assign '4' to a constant so that I can be found in a code search?